### PR TITLE
Include legacy documentation in pdf and MS Word format - Modules

### PR DESCRIPTION
**Pull request by rafmudaf**
**Feature or improvement description**
Until the legacy documentation can be fully converted to the readthedocs / sphinx format, we will be including old documentation in the existing format.

This pull request includes documentation for specific modules and FAST.Farm. It is a follow on to #805.

**Related issue, if one exists**
#804

**Impacted areas of the software**
Online docs.

See the documentation live at https://raf-openfast.readthedocs.io/en/docs-legacy/source/user/index.html

### CODE REVIEW

**Comment by ebranlard** [comment]
Sorry if this is already included, but it would be nice to add some links for the simulink interface. I think the main documentation is in the FAST 8 pdf document. But we can also point to how to compile it, and where the examples are in the repository. I did receive a bunch of questions about this lately. Sorry if it's already included. Here's the info I was provided with:

The FAST v8 User's Guide provides some documentation on the Simulink interface (https://www.nrel.gov/wind/nwtc/assets/downloads/FAST/FASTv8.16/README_FAST8.pdf).

The above FAST v8 guide can be used with the code and examples here:
https://github.com/OpenFAST/openfast/tree/dev/glue-codes/simulink

File: docs/source/user/index.rst [hunk 1]
```diff
@@ -78,3 +78,53 @@ Glue Code and Mesh Mapping
...
+
+
+NWTC Subroutine Library
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- :download:`NWTC Library – short overview of subroutines and functions <../../OtherSupporting/NWTC_Library_Description.pdf>`
+
+
+Other Modules
+~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (rafmudaf) [reply docs/source/user/index.rst:90]: I consolidated these two sections since they're both "module documentation"

**Comment by rafmudaf** [comment]
@jjonkman I've addressed your comments here and merged the latest `dev` branch

File: docs/source/user/beamdyn/index.rst [hunk 2]
```diff
@@ -1,19 +1,21 @@
 BeamDyn User Guide and Theory Manual
 ======================================
 
-.. only:: html
+This document offers a quick reference guide for the BeamDyn software
+program. It is intended to be used by the general user in combination
+with other FAST manuals. The manual will be updated as new releases are
```
Comment (jjonkman) [inline docs/source/user/beamdyn/index.rst:6]: Change "FAST" to "OpenFAST".

**Review by jjonkman — APPROVED** [review]
Thanks for addressing my comments from my original review.  I now approve this merge.  Please find one minor change below.

**Pull request merged**